src/components/ThoughtAnnotation.js

```diff
@@ -20,12 +19,16 @@ import {
   pathToContext,
   unroot,
 } from '../util'
+import { store } from '../store'
 
 // components
 import HomeLink from './HomeLink'
 import StaticSuperscript from './StaticSuperscript'
 import ContextBreadcrumbs from './ContextBreadcrumbs'
 
+// selectors
+import { decodeThoughtsUrl } from '../selectors'
+
 const mapStateToProps = ({ cursor, cursorBeforeEdit, focusOffset, invalidState, editingValue }, props) => {
```

Comment:
This can be replaced with
```
const mapStateToProps = (state, props) => {
    const { cursor, cursorBeforeEdit, focusOffset, invalidState, editingValue } = state
```